chain/client/src/client_actor.rs

```diff
@@ -1202,6 +1204,19 @@ impl ClientActor {
 
         let timer = metrics::CHECK_TRIGGERS_TIME.start_timer();
         if self.sync_started {
+            self.sync_timer_next_attempt = self.run_timer(
+                self.sync_wait_period(),
+                self.sync_timer_next_attempt,
+                ctx,
+                |act, _| act.run_sync_step(),
+                "sync",
+            );
+
+            delay = core::cmp::min(
```

Comment:
```suggestion
            delay = std::cmp::min(
```

maybe just `delay = delay.min(self.sync_timer_next_attempt.signed_duration_since(now).to_std().unwrap_or(delay))`? that feels like a neater way to write `min=`